### PR TITLE
animation from images

### PR DESCRIPTION
**Pull request by Omid07**

### CODE REVIEW

File: app/src/main/java/com/example/omid/ps01/MainActivity.java [hunk 1]
```diff
@@ -84,32 +97,50 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
...
+                intent.putExtra(getString(R.string.images_path), paths);
+                startActivity(intent);
+            } else {
+                Toast.makeText(getApplicationContext(), R.string.select_two_images, Toast
+                        .LENGTH_LONG).show();
+            }
+        }
+        if (requestCode == Constant.PICK_IMAGES_FOR_VIDEO) {
+            ArrayList<String> paths = new ArrayList<>();
+            paths = getImagesPaths(data);
```
Comment (b1acKr0se) [inline app/src/main/java/com/example/omid/ps01/MainActivity.java]: Initialize variable and assign value with one line

File: app/build.gradle [hunk 2]
```diff
@@ -25,4 +25,5 @@ dependencies {
     compile 'com.android.support:appcompat-v7:23.4.0'
     compile 'com.android.support:support-v4:23.4.0'
     compile 'com.theartofdev.edmodo:android-image-cropper:2.2.+'
+    compile 'com.google.guava:guava:19.0'
```
Comment (b1acKr0se) [inline app/build.gradle]: what is it for?

Comment (Omid07) [inline app/build.gradle]: @b1acKr0se  i use this library to initialize variable and assign value with one line.

Comment (Omid07) [inline app/build.gradle]: @b1acKr0se  i use this library to Initialize variable and assign value with one line in an arrayList

Comment (b1acKr0se) [inline app/build.gradle]: @Omid07 what's wrong with
` List<String> paths = getImagesPaths(data);`

Comment (Omid07) [inline app/build.gradle]: @b1acKr0se  yes..i have already changed it..i didn't notice it..

File: app/src/main/java/com/example/omid/ps01/MainActivity.java [hunk 3]
```diff
@@ -83,33 +96,49 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
...
+                Intent intent = new Intent(MainActivity.this, MergeImageActivity.class);
+                intent.putExtra(getString(R.string.images_path), paths);
+                startActivity(intent);
+            } else {
+                Toast.makeText(getApplicationContext(), R.string.select_two_images, Toast
+                        .LENGTH_LONG).show();
+            }
+        }
+        if (requestCode == Constant.PICK_IMAGES_FOR_VIDEO) {
+            ArrayList<String> imagesPaths= getImagesPaths(data);
```
Comment (b1acKr0se) [inline app/src/main/java/com/example/omid/ps01/MainActivity.java]: please format this file

**Comment by b1acKr0se** [comment]
OK

**Comment by nvmanh** [comment]
OK!

**Pull request merged**